docs/tutorial/index.rst

```diff
@@ -0,0 +1,270 @@
+Read the Docs tutorial
+======================
+
+In this tutorial you will create a documentation project on Read the Docs
+by importing an Sphinx project from a GitHub repository,
+tailor its configuration, and explore several useful features of the platform.
+
+The tutorial is aimed at people interested in learning
+how to use Read the Docs to host their documentation projects.
+You will fork a fictional software library
+similar to the one developed in the :doc:`official Sphinx tutorial <sphinx:tutorial/index>`.
+No prior experience with Sphinx is required,
+and you can follow this tutorial without having done the Sphinx one.
```

Comment:
This reads nicely, and gives the user somewhere to learn more if needed. Love it 💯